build.gradle

```diff
@@ -41,6 +41,7 @@ repositories {
 dependencies {
     implementation 'net.i2p.crypto:eddsa:0.3.0'
     testImplementation 'org.junit.jupiter:junit-jupiter:5.7.0'
+    testImplementation 'org.hamcrest:hamcrest:2.2'
```

Comment:
can you do this without hamcrest. There is a strict policy to not use any libraries, even for test

Reply:
Ya no problem. I guess I figured that hamcrest is on the same "level" as junit in terms of ubiquitous usage.